### DISCUSSION TITLE
欢迎来到讨论区 | Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by shengxinjing** [tutorial-kit/community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 3000-codes** [comment]
什么时候,什么平台开课呢
